Gemfile

```
@@ -77,7 +77,7 @@ gem 'devise-i18n'
 gem 'devise-guests', '~> 0.6'
 gem 'omniauth'
 gem 'omniauth-cas'
-gem 'ldap_groups_lookup', '~> 0.4.0'
+gem 'ldap_groups_lookup', '~> 0.5.0'
```

```suggestion
gem 'ldap_groups_lookup', '~> 0.6.0'
```
I bumped the upstream gem version.